src/main/java/me/figo/FigoSession.java

```diff
@@ -270,46 +165,9 @@ public LoginSettings getLoginSettings(String countryCode, String bankCode) throw
     public TaskTokenResponse setupNewAccount(String bankCode, String countryCode, String loginName, String pin) throws FigoException, IOException	{
     	return this.queryApi("/rest/accounts", new SetupAccountRequest(bankCode, countryCode, loginName, pin), "POST", TaskTokenResponse.class);
     }
-    
-    /**
-     * Setups an account an starts the initial syncronization directly
-     * @param bankCode
-     * @param countryCode
-     * @param loginName
-     * @param pin
-     * @return TaskStatusResponse
-     */
-    public TaskStatusResponse setupAndSyncAccount(String bankCode, String countryCode, String loginName, String pin) throws FigoException, IOException, FigoPinException, InterruptedException	{
```

Comment:
I am honestly puzzled by this code myself, but it is an important feature (although very badly implemented). Can we either clean it up or mark it as dangerous instead of removing?